srcs/params/ibvFunction.cpp

```diff
@@ -75,7 +75,7 @@ void reflectShallow(std::vector<double>& uAtIBC, const std::vector<double>& pos,
 {
 
     // check that there is enough values
-    assert(u.size() == uAtIBC.size());
+    assert(field.u.size() == uAtIBC.size());
```

Comment:
We should remove those assets because they are wrong. `uAtIBC.size()` is 3 here, while `field.u.size()` is HUGE.